src/components/Head.astro

```diff
@@ -1,9 +1,8 @@
 ---
-import type { Props } from "@astrojs/starlight/props";
 import Default from "@astrojs/starlight/components/Head.astro";
 ---
 
-<Default {...Astro.props}><slot /></Default>
```

Comment:
Can you explain the rational for the removal of the Astro.props changes generally in this? Perhaps add a link to the changelog section where this change is mentioned that would help someone in future understanding why this changes.

Reply:
Got it. Thanks for the review!